src/catalog/src/durable/impls/persist.rs

```diff
@@ -124,7 +124,7 @@ impl UnopenedPersistCatalogState {
     }
 
     /// Create a new [`UnopenedPersistCatalogState`] to the catalog state associated with `organization_id`.
-    #[tracing::instrument(level = "debug", skip(persist_client))]
+    #[tracing::instrument(level = "info", skip(persist_client))]
```

Comment:
Also probably want to skip metrics here.

In general, I've been doing skip_all and then allowlisting things, because the failure case is less bad, but that'd be pretty invasive here, so up to you

Reply:
Done. I think I'll keep it as is for now and maybe update to `skip_all` in a follow up.